invenio_records_resources/services/custom_fields/mappings.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Custom fields mappings."""
```

Comment:
Note: the search engine and version information could be imported here (i.e. `SEARCH_DISTRIBUTION` from invenio-search) and used to calcualte based on it.